gpu-jobs.md

```diff
@@ -308,18 +286,44 @@ runtime is installed (or there's no runtime at all!). For compiled code,
 look for flags that enable static linking or use one of the solutions
 listed below.
 
-## B. Docker
+## 2. Docker
 
 CHTC's GPU servers have "nvidia-docker" installed, a specific version of
 Docker that integrates Docker containers with GPUs. If you can find or
 create a Docker image with your software that is based on the
 nvidia-docker container, you can use this to run your jobs in CHTC. See
 our [Docker guide](/docker-jobs.shtml) for how to use Docker in CHTC.
 
-# 4. Using GPUs on the Open Science Grid
+
+# D. Access Additional GPU Resources
```

Comment:
I could imagine some confusion about whether the special GPU Lab policies apply to jobs in this category.  Should we be explicit about which instructions above are still relevant?  For instance, the GPU job types are no longer needed here.  `request_gpus = 1` is still needed in the submit file, but the other GPU Lab-specific lines are not.